src/sse/chainweb-cut.ts

```diff
@@ -8,13 +8,15 @@ const CLASS_NAME = 'ChainwebCutService'
 export default class ChainwebCutService {
   logger = new Logger('CutService');
   lastCut: ChainwebCutData;
+  hasCut: Promise<boolean>;
   lastUpdateTime;
   running = false;
 
   _config;
   _updateInterval;
   _intervalId;
   _updateCallbacks = new Set<ChainwebCutCallback>();
+  _resolveFirstData: Function;
```

Comment:
Use the sig for the actual function (e.g. `(someArg: type) => returnvalue`)

Reply:
Also, probably good to use `@kadena-dev/eslint-config` in this package, and extend from the `lib` variant. See [this `./eslintrc` config](https://github.com/kadena-community/kadena.js/blob/main/packages/libs/client/.eslintrc.js) for an example